pinot-common/src/main/java/org/apache/pinot/common/config/IndexingConfig.java

```diff
@@ -145,6 +146,14 @@ public void setOnHeapDictionaryColumns(List<String> onHeapDictionaryColumns) {
     _onHeapDictionaryColumns = onHeapDictionaryColumns;
   }
 
+  public boolean isEnableDefaultStarTree() {
+    return _enableDefaultStarTree;
+  }
+
+  public void setEnableDefaultStarTree(boolean enableDefaultStarTree) {
+    _enableDefaultStarTree = enableDefaultStarTree;
```

Comment:
Another way to add this config may be to add a "mode" field in StarTreeIndexConfig. If mode is set to "auto" then ignore other settings. Otherwise, follow the other serttings.

Reply:
I prefer the current way so that the default star-tree can be enabled by one boolean flag. Don't want to change the existing StarTreeIndexConfig because adding a mode seems more confusing to me.